manifests/capacityscheduling/eq1.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: capacityscheduling.sigs.k8s.io/v1alpha1
```

```suggestion
apiVersion: scheduling.sigs.k8s.io/v1alpha1
```

And name this file as "elasticquota.yaml"

---

Sure I rename it as `elasticquota-example.yaml`